### DISCUSSION TITLE
using out-of-the-box

### DISCUSSION THREAD

**Discussion by tcooperma** [vmware/packer-examples-for-vsphere#971] [Help] [answered]
I get the error creating a VM:

"ServerFaultCode: The operation is not supported on the object."

It looks like on my VMware server it says it is trying to create a folder and cannot do it.

I am on VMware 8.0, the latest version of the packer-plugin-vsphere, and packer v1.11.2

**Comment by tenthirtyam** [comment]
Are you running it against a standalone ESXi host instead of a vCenter Server instance?

**Comment by tcooperma** [comment]
Yes, I do not have Vcenter set up. should I set up Vcenter to play with it?  
The previous examples of using it with the ESXi are also failing with the same error.  I had an example with RHEL8 which previously worked and not fail with the same error.I assume something changed with the vmware packer plugin?

    On Thursday, September 26, 2024 at 12:45:36 PM EDT, Ryan Johnson ***@***.***> wrote:  
 
 


Are you running it against a standalone ESXi host instead of a vCenter Server instance?

**Comment by tenthirtyam** [comment, marked as answer]
This examples project is designed to work with vCenter Server instead of direct ESXi.

It can be adjsted to work with direct ESXi but it's a bit of effort - commenting out folder, cluster, datacenter, and the use of the content library for each build definition.

**Comment by tcooperma** [comment]
That worked, thanks.  I am having a problem with Ubuntu server, but Fedora and Rocky seem to be working now that I am using vCenter.

    On Thursday, September 26, 2024 at 01:09:31 PM EDT, Ryan Johnson ***@***.***> wrote:  
 
 


This examples project is designed to work with vCenter Server instead of direct ESXi.

It can be adjsted to work with direct ESXi but it's a bit of effort - commenting out folder, cluster, datacenter, and the use of the content library for each build definition.
